allensdk/internal/brain_observatory/time_sync.py

```diff
@@ -18,37 +18,43 @@
 LONG_STIM_THRESHOLD = 0.2     # seconds
 MAX_MONITOR_DELAY = 0.07     # seconds
 
-VERSION_1_KEYS = {
-    "photodiode": "stim_photodiode",
-    "2p": "2p_vsync",
-    "stimulus": "stim_vsync",
-    "eye_camera": "cam2_exposure",
-    "behavior_camera": "cam1_exposure",
-    "acquiring": "2p_acquiring",
-    "lick_sensor": "lick_1"
-    }
-
-# MPE is changing keys. This isn't versioned in the file.
-VERSION_2_KEYS = {
-    "photodiode": "photodiode",
-    "2p": "2p_vsync",
-    "stimulus": "stim_vsync",
-    "eye_camera": "eye_tracking",
-    "behavior_camera": "behavior_monitoring",
-    "acquiring": "2p_acquiring",
-    "lick_sensor": "lick_sensor"
-    }
-
 
 def get_keys(sync_dset):
     """Get the correct lookup for line labels.
 
     This method is fragile, but not all old data contains the full list
```

Comment:
I think now that you raise an exception, this method is not fragile. I'm not even sure what that means,

Reply:
I didn't write this comment

Reply:
i know. but you're cleaning up a not-great or out-of-date function. now that it is improved, why retain the old description of "I'm not great"?

Reply:
Makes sense, replaced with more informative doc string

Reply:
@djkapner should resolve this conversation